drivers/postmanApiLoadDriver/DockerFile

```diff
@@ -0,0 +1,22 @@
+FROM centos:centos7
```

Comment:
Why do we need a separate DockerFile for this? Can we not run the `newman run` command in the torpedo pod itself?
I think we just need to add the newman utility to existing torpedo image.

Reply:
As suggested, I have added the newman dependencies in torpedo DockerFile and have wriiten a new function to just run the newman command in torpedo pod.
Howerver I would suggest not to remove "run newman as a separate pod" mechanism for future use if need be.